### PR TITLE
fix panic due to nil route dst

### PR DESCRIPTION
**Pull request by cyclinder**
## Related issues/PRs

https://github.com/projectcalico/calico/issues/7645

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
TBD
```

## Reminder for the reviewer

Make sure that this PR has the correct labels and milestone set.

Every PR needs one `docs-*` label.

- `docs-pr-required`: This change requires a change to the documentation that has not been completed yet.
- `docs-completed`: This change has all necessary documentation completed.
- `docs-not-required`: This change has no user-facing impact and requires no docs.

Every PR needs one `release-note-*` label.

- `release-note-required`: This PR has user-facing changes. Most PRs should have this label.
- `release-note-not-required`: This PR has no user-facing changes.

Other optional labels:

- `cherry-pick-candidate`: This PR should be cherry-picked to an earlier release. For bug fixes only.
- `needs-operator-pr`: This PR is related to install and requires a corresponding change to the operator.

### CODE REVIEW

File: felix/ifacemonitor/iface_monitor.go [hunk 1]
```diff
@@ -219,6 +219,11 @@ func (m *InterfaceMonitor) handleNetlinkRouteUpdate(update netlink.RouteUpdate)
 		return
 	}
 
+	// issue: https://github.com/projectcalico/calico/issues/7645
```
Comment (mazdakn) [inline felix/ifacemonitor/iface_monitor.go:222]: Can you please remove this comment. By using `git blame` and checking the PR, it's possible to get back to the original issue.

**Review by mazdakn — CHANGES_REQUESTED** [review]
LGTM. Just please remove the reference to the GH issue, since we can find it anyway.

File: felix/ifacemonitor/iface_monitor.go (hunk 1 above)
Comment (cyclinder) [reply felix/ifacemonitor/iface_monitor.go]: Thanks for the review @mazdakn , Now updated.

**Comment by mazdakn** [comment]
@cyclinder can you please also create the back port PRs for `v3.26`, `v3.25` and `v3.24`? If not, I can make those.

**Comment by cyclinder** [comment]
Yes, sure. @mazdakn , Maybe I can do this in this weekend.

**Comment by cyclinder** [comment]
@mazdakn , all back port PRs has been created, Can you take a loook?

v3.24: https://github.com/projectcalico/calico/pull/7710
v3.25: https://github.com/projectcalico/calico/pull/7709
v3.26: https://github.com/projectcalico/calico/pull/7711

**Pull request merged**